javascript/js-modern-syntax/js-modern-syntax_destructuring-objects/index.js

```diff
@@ -49,17 +57,22 @@ const logInfo = (city) => {
   const country = city.country;
   const numPeople = city.population;
 
-  return `${name} is in ${country} and has ${numPeople} inhabitants in it.`
-}
+  return `${name} is in ${country} and has ${numPeople} inhabitants in it.`;
+};
+
+const { name, country, numPeople } = city;
 
 const destructuringLogInfo = (city) => {
   // (Hint: Lines 48-50 should be replaced with a single line and you shouldn't change anything else).
   // your code here
 
-
   // your code here
-  return `${name} is in ${country} and has ${numPeople} inhabitants in it.`
-}
+  return `${name} is in ${country} and has ${numPeople} inhabitants in it.`;
```

Comment:
Here goes: `const { name, country, population: numPeople } = city` as I mentioned above, so that the return statement is moved to line 71.